v2/gcs-to-sourcedb/src/test/java/com/google/cloud/teleport/v2/templates/GCSToSourceDbInterleaveIT.java

```diff
@@ -169,7 +169,7 @@ private void assertInsertedRowsInMySQL() throws InterruptedException {
     PipelineOperator.Result result =
         pipelineOperator()
             .waitForCondition(
-                createConfig(writerJobInfo, Duration.ofMinutes(10)),
+                createConfig(writerJobInfo, Duration.ofMinutes(30)),
```

Comment:
Should we pass the dlqRetryInterval minutes to something less than 10 mins, instead of increasing the timeout by so much?

Reply:
After digging deeper, I found the dlq retry minutes are only used when running in retry mode, where it polls directly on gcs. In normal mode, there is no retry configured. The moment a notification is available in DLQ topic, it consumes.